site/src/docs/foundation/design-tokens/colour/index.mdx

```diff
@@ -76,12 +78,13 @@ Brand colours also have dark, medium light and light variants. You can find thes
-| Success                  | Positive state to inform user about success.                                                            | _"Form submit was successful!"_             | <ColorExample color="var(--color-success)" name="Success status colour example" /> |
-| Alert                    | Warning state to catch users attention but does not require immediate actions.                          | _"Loading is taking longer than expected."_ | <ColorExample color="var(--color-alert)" name="Alert status colour example" />     |
-| Error                    | Error state which requires users immediate action.                                                      | _"Form is missing critical information."_   | <ColorExample color="var(--color-error)" name="Error status colour example" />     |
+| Status name              | Usage                                                                                                   | Example message                             | Example                                                                                                |
+| ------------------------ | ------------------------------------------------------------------------------------------------------- | ------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| Info                     | State which does not indicate clear success or failure but can be useful extra information to the user. | _"You have received new messages."_         | <ColorExample color="var(--color-info)" name="Info status colour example" />                           |
+| Success                  | Positive state to inform user about success.                                                            | _"Form submit was successful!"_             | <ColorExample color="var(--color-success)" name="Success status colour example" />                     |
+| Alert                    | Warning state to catch users attention but does not require immediate actions.                          | _"Loading is taking longer than expected."_ | <ColorExample color="var(--color-alert)" name="Alert status colour example" />                         |
+| Error                    | Error state which requires users immediate action.                                                      | _"Form is missing critical information."_   | <ColorExample color="var(--color-error)" name="Error status colour example" />                         |
+| Focus                    | Focus colour adds blue outline around the focused element.                                              |                                             | <FocusColorExample color="var(--color-focus-outline)" contentStyle={{height: '28px', width: '28px'}}/> |
```

Comment:
Maybe alter this usage text so it's more in the way of explaining why to use it? Like the states above are explained. Maybe something like "Indication for active selected element which helps users navigating the page with a keyboard."